### ISSUE TITLE
React 17/18 Support

### ISSUE THREAD

**Issue by dhalucario** [zeroasterisk/react-iframe-resizer-super#25]
Hey, I was wondering if this component works with React 17/18 and if we could update the `package.json` to reflect that.

**Comment by danny-faith** [comment]
Hello, any movement on this? thanks 0_0
@dhalucario hey mate, did you get anywhere with React 17/18 and this package?

**Comment by davidjbradshaw** [comment]
I don’t think this package is maintained anymore. Have a look at https://iframe-resizer.com/react
